src/v2/Components/__tests__/LotTimer.jest.tsx

```diff
@@ -118,7 +118,7 @@ describe("extendedBiddingInfoCopy", () => {
         }
         const wrapper = mount(<LotTimer saleArtwork={saleArtwork} />)
         const text = wrapper.text()
-        expect(text).toContain("Extended: 2m 0s")
+        expect(text).toContain("Extended:")
```

Comment:
no, this has come up before. we can and should be making specific assertions about dates (or relative times) in specs. Checking `Extended:` without the timestamp is just not a great spec (its ok and better than nothing, but it's not great).

if something is flaky w.r.t to running, then rather than use the current time, the current time itself should be mocked to a specific time (here, maybe it's the `useTimer` hook or whatever).

It's akin to your earlier PR in metaphysics- https://github.com/artsy/metaphysics/pull/3862/files#diff-ee0c6484114d191bc503477b20472ce84762c12795409eed2d07d57ad7f2ce78R886

Reply:
If this is something we expect as a practice, [should we document it in the best practices doc](https://github.com/artsy/force/blob/main/docs/best_practices.md#write-unit-tests-for-new-components)?

Maybe we can find an example of where this is done well in a Force test for knowledge sharing purposes (cc @damassi do you know of any good time-related examples)?